frontend-build-pipeline.yml

```diff
@@ -16,27 +19,48 @@ parameters:
 - name: sonarCloudProjectName # sonar cloud's cli project name
   type: string
   default: 'RevatureShopFrontend'
+- name: sonarCloudExclude # files for sonar cloud to ignore
```

Comment:
* to ignore specifically from coverage calculations, not from everything else.

Reply:
I must have misunderstood when I made the parameter, updated comment and changed the name to sonarCloudCoverageExclude.